assets/terraform-modules/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -270,7 +270,11 @@ storage:
       contents:
         inline: |
           {
-            "live-restore": true
+            "live-restore": true,
+            "log-opts": {
+              "max-size": "100m",
```

Comment:
Why 100m and 3 (and not 200m and 2)?

We could also have some note in the documentation, that logs will be limited to last 300MB for each container.

Reply:
No specific reason to keep it to `100m x 3` vs `200m x 2`. A default to start with, is all.

Reply:
I think we could make some assumptions here. Let's say minimal disk size of 20GB and max number of pods per node on 2GB or 4GB machine and assume they all log extensively.

Reply:
Assuming that 20GB is exclusively available to store logs:

- With 300MB limit we can fit 66 containers,
- With 400MB limit we can fit 50 containers.

Reply:
Hm, default for `--max-pods` is 110. I wonder how that correlates.

Reply:
We documented the constraints, so marking as resolved.